src/main/java/appeng/core/definitions/AEBlocks.java

```diff
@@ -112,14 +112,15 @@ public final class AEBlocks {
     private static final List<BlockDefinition<?>> BLOCKS = new ArrayList<>();
     private static final Properties QUARTZ_CLUSTER_PROPERTIES = defaultProps(MapColor.COLOR_CYAN,
             SoundType.AMETHYST_CLUSTER).forceSolidOn().strength(1.5f).requiresCorrectToolForDrops();
-    private static final Properties QUARTZ_PROPERTIES = stoneProps()
+    private static final Properties QUARTZ_PROPERTIES = stoneProps().mapColor(MapColor.COLOR_CYAN)
             .strength(3, 5).requiresCorrectToolForDrops();
-    private static final Properties SKYSTONE_PROPERTIES = stoneProps()
+    private static final Properties SKYSTONE_PROPERTIES = stoneProps().mapColor(MapColor.COLOR_BLACK)
             .strength(5, 150).requiresCorrectToolForDrops();
     private static final StateArgumentPredicate<EntityType<?>> NEVER_ALLOW_SPAWN = (p1, p2, p3,
             p4) -> false;
-    private static final Properties SKY_STONE_CHEST_PROPS = stoneProps()
+    private static final Properties SKY_STONE_CHEST_PROPS = stoneProps().mapColor(MapColor.COLOR_BLACK)
             .strength(5, 150).noOcclusion();
+    private static final Properties FLUIX_PROPERTIES = QUARTZ_PROPERTIES.mapColor(MapColor.COLOR_PURPLE);
```

Comment:
This will mutate the quartz properties instead of deriving a separate properties instance with purple map color from it

Reply:
I hadn't accounted for that for whatever reason, my bad.